packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1368,6 +1368,11 @@ const schema: SchemaType<"Users"> = {
   notificationSubscribedUserPost: {
     label: "Posts by users I'm subscribed to",
     ...notificationTypeSettingsField(),
+    onCreate: () => {
+      if (!isLWorAF) {
+        return {...defaultNotificationTypeSettings, channel: 'both'}
+      }
+    }
```

Comment:
I'm pretty sure that we are still not allowed to actually have different default values for fields, so I did it this way instead